buildSrc/src/main/java/org/elasticsearch/gradle/ElasticsearchDistribution.java

```diff
@@ -283,7 +289,7 @@ void finalizeValues() {
                     "platform cannot be set on elasticsearch distribution [" + name + "] of type [" + getType() + "]"
                 );
             }
```

Comment:
We should actually add some validation in here so you can't choose `DOCKER_UBI` along with the `OSS` flavor.

Reply:
I've added a check here.